<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gpt_show_session</class>
 <widget class="QWidget" name="gpt_show_session">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查看会话</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
